apps/console/src/features/applications/data/application-templates/templates/single-page-application/create-wizard-help.tsx

```diff
@@ -57,11 +57,12 @@ const SinglePageApplicationCreateWizardHelp: FunctionComponent<SinglePageApplica
             <React.Fragment>
                 <Heading as="h5">Allowed Redirect URLs</Heading>
                 <p>
-                    The redirect URL determines to where the authorization code is sent, login is
-                    completed and to where the user is redirected to once the logout is completed.
+                    The redirect URL determines where the authorization code is sent to
+                    once the user is authenticated, and where the user is redirected to
+                    once the logout is completed.
                 </p>
-                <p>The client should specify the redirect_url in the authorize or logout request and
-                    &nbsp;{ config.ui.productName }&nbsp;will validate it against this configured redirect URLs.
+                <p>The redirect URLs sent by the client application along with the authentication
```

Comment:
The spec usually calls these endpoints as redirect URIs.
Can we have a quick search on this and be consistent with the correct terminology?

This is how Google displays it

![image](https://user-images.githubusercontent.com/5237653/105847171-4395ef00-6003-11eb-9d2c-fc02f0f637f9.png)

Reply:
`Allowed Redirect URLs` is renamed as `Authorized redirect URIs` with https://github.com/wso2/identity-apps/pull/1580